packages/tools/src/utilities/math/polyline/isPointInsidePolyline.ts

```diff
@@ -0,0 +1,98 @@
+import type { Types } from '@cornerstonejs/core';
+
+/**
+ * Determines whether a 2D point is inside a polyline.
```

Comment:
Please indicate the boundary conditions - what happens if a point is on the boundary?  I believe that should be considered inside, but with the naming/description, I just don't know.